content/60.methods.md

```diff
@@ -2,8 +2,24 @@
 
 ### ABSTRACT
```

Comment:
Does ACM have a word limit on abstracts? I couldn't find it. This is 363 words which seems too long, but if they don't say otherwise...

Reply:
The sample manuscript doesn't state a word limit either https://github.com/greenelab/covid19-review/blob/c3bf3f4245cc4fec4c37f349184015e1eafdacfc/build/assets/acmart-master/samples/sample-sigconf.tex#L169